### PR TITLE
Improve artifact

### PR DESCRIPTION
**Pull request by Felalolf**
We later need to change the GOBRA_COMMIT arg to a tag.

### CODE REVIEW

File: artifact/content/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,681 @@
...
+(*counter) implements stream {
+  ...
+}
+```
+
+- Explained in [the section on interfaces](#interfaces)
+
+## Basic Annotations
+
+All examples shown in this tutorial can be found [here](../src/test/resources/regressions/examples/tutorial-examples/). We start with a simple function `sum` computing the sum of the integers from 0 to `n`. The table below shows a Go implementation (left) and a corresponding annotated version (right).
```
Comment (jcp19) [inline artifact/content/tutorial.md:110]: now that the `tutorial-examples` is in the master branch, we can put a link to it. If we don't change it for this particular copy of the tutorial, the link will not work
```suggestion
All examples shown in this tutorial can be found [here](https://github.com/viperproject/gobra/tree/master/src/test/resources/regressions/examples/tutorial-examples). We start with a simple function `sum` computing the sum of the integers from 0 to `n`. The table below shows a Go implementation (left) and a corresponding annotated version (right).
```

Comment (ArquintL) [reply artifact/content/tutorial.md]: Should this be eventually changed to the `artifact1` tag as well?

**Review by ArquintL — APPROVED** [review]
As this PR contains my commits, I'm obviously approving it ^^

File: artifact/content/tutorial.md (hunk 1 above)
Comment (Felalolf) [reply artifact/content/tutorial.md]: yes

**Pull request merged**